site/zh/blog/new-in-devtools-107/index.md

```diff
@@ -27,94 +27,97 @@ draft: true
 
 <!-- Content starts here -->
 
-<!-- ## Customize keyboard shortcuts in DevTools {: #shortcuts } -->
+## 客制化 DevTools 的键盘快捷键 {: #shortcuts }
 
-<!-- You can now customize keyboard shortcuts for your favorite commands in DevTools. -->
+您现在可以在 DevTools 里客制化您喜爱的键盘快捷键。
 
-<!-- Go to **Settings** > **Shortcuts**, hover over a command and click the **Edit** button (pen icon) to customize the keyboard shortcut. You can create chords (a.k.a multi-key press shortcuts) as well.  -->
+去到 **Settings（设置）** > **Shortcuts（快捷键）**, 将鼠标徘徊在命令上并点击 **Edit（编辑）** 按钮 (钢笔图示）来客制化键盘快捷键。您也可以创建组合键（a.k.a. 多键点击快捷键）。
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/973EfWpxwGOdEF1nN1vv.png", alt="Customize keyboard shortcuts in DevTools.", width="800", height="516" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/973EfWpxwGOdEF1nN1vv.png", alt="客制化 DevTools 的键盘快捷键。", width="800", height="516" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/d061128ff63a97ab2c6c0d2b5e655e6fcbed829c #}
 
-Chromium issues: [1335274](https://crbug.com/1335274), [174309](https://crbug.com/174309)
+Chromium 议题: [1335274](https://crbug.com/1335274), [174309](https://crbug.com/174309)
 
 
-<!-- ## Toggle light and dark themes with keyboard shortcut {: #toggle-themes } -->
+## 使用键盘快捷键对浅色与深色主题进行快速切换。 {: #toggle-themes }
+
+设置一个键盘快捷键来对浅色与深色主图便利地进行快速切换。这个行动的快捷键并没有被系统预设。 
```

Comment:
```suggestion
```

This line is not suppose to be here, always translate under the original English sentence